cli/pkg/local/app.go

```diff
@@ -210,9 +210,6 @@ func (a *App) Reconcile() error {
 		a.Logger.Errorf("Hydration canceled")
 		return nil
 	}
-	for _, path := range res.AffectedPaths {
-		a.Logger.Infof("Reconciled: %s", path)
-	}
```

Comment:
Let's keep this as it will come after the reconciliation process and show all impacted paths (including models and metrics views)